src/eduid_webapp/email/views.py

```diff
@@ -227,15 +236,14 @@ def verify_link(user):
         if code != state.verification.verification_code:
             current_app.logger.warning("Invalid verification code for: {}".format(state.verification.email))
             return redirect(current_app.config['SAML2_LOGIN_REDIRECT_URL'])
-
         try:
             verify_mail_address(state, proofing_user)
+            current_app.logger.info('Verified email {} for user {!r}'.format(email, user))
+            return redirect(current_app.config['SAML2_LOGIN_REDIRECT_URL'])
```

Comment:
This return looks redundant with the one after the try: clause if you were to undo the indent-changes of the one you moved into the except: ?

Reply:
fixed